### PR TITLE
Accessibility addition

### PR DESCRIPTION
**Pull request by princebaretto99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
-# ios-sample-app
+# XCUITests with A11yUITests for Accessibility Testing on iOS devices
 
 A sample iOS app supporting [XCUI Tests](https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/09-ui_testing.html) on [Browserstack](https://www.browserstack.com/).
 
 <img src="https://cdn-images-1.medium.com/max/1600/1*Z0AH-kvjNsUKlcgjP01rmA.png" height="120" /> ![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
 
 ## How to run
```
Comment (Sankalan1987) [inline README.md]: Document all the Pre-requisites like
1. Xcode installation
2. Cocoapods installation
3. Mention Developer Certificate required for iOS App Build.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
+```
+	pod install
+```
+
 1. Select the device as "Generic iOS device"
 2. Product -> Clean
```
Comment (Sankalan1987) [inline README.md]: Please make following changes : 
Product -> Clean Build
Product -> Clean Test

File: README.md (hunk 1 above)
Comment (Sankalan1987) [inline README.md]: Change the Header as Pre-requisites instead How to run

File: README.md [hunk 3]
```diff
@@ -21,3 +25,5 @@ A sample iOS app supporting [XCUI Tests](https://developer.apple.com/library/con
 	5. zip -r SampleUITests.zip SampleXCUITests-Runner.app/
```
Comment (Sankalan1987) [inline README.md:34]: These steps can be covered under Run Tests on BrowserStack App Automate section

File: SampleXCUITests/SampleXCUITests.swift [hunk 4]
```diff
@@ -62,5 +65,6 @@ class SampleXCUITests: XCTestCase {
         
         //verify that the text entered matches the text view
         XCTAssertEqual(app.staticTexts.element.label, enterText)
+        a11yCheckAllOnScreen()
```
Comment (Sankalan1987) [inline SampleXCUITests/SampleXCUITests.swift]: Add few more A11yUI validation specific to Buttons, StaticTexts, Minimum Size etc.

File: README.md [hunk 5]
```diff
@@ -21,3 +25,5 @@ A sample iOS app supporting [XCUI Tests](https://developer.apple.com/library/con
...
 5. Use the ipa, and zip file to run on Browserstack
 
+## Note : Follow [This](https://www.browserstack.com/docs/app-automate/xcuitest/getting-started) for executing the tests on BrowserStack
+
```
Comment (Sankalan1987) [inline README.md:51]: Include a section Links & Resources in Readme and provide the link https://github.com/rwapp/A11yUITests & https://developer.apple.com/documentation/xctest

**Review by Sankalan1987 — CHANGES_REQUESTED** [review]
@princebaretto99 : Please incorporate the review feedbacks and send it for review.

File: README.md (hunk 1 above)
Comment (princebaretto99) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (princebaretto99) [reply README.md]: Done

**Review by Sankalan1987 — APPROVED** [review]
Changes look great. Approved the PR.

**Pull request merged**